libmscore/score.cpp

```diff
@@ -2295,11 +2295,19 @@ bool Score::appendScore(Score* score)
             return false;
       int tickOfAppend = lastMeasure->endTick();
 
-      if (!lastMeasure->lineBreak() && !lastMeasure->pageBreak()) {
-            lastMeasure->undoSetBreak(true, LayoutBreak::Type::LINE);
+      if (addPageBreak) {
+            // add PageBreak if one doesn't already exist
+            if (!lastMeasure->pageBreak())
```

Comment:
So score that have a line break at their end, where this presumably has been added on purpose, don't get a page break. Yes, sounds good to me.